Tests/UnitTests/Misc/TimingUtilTests.swift

```diff
@@ -14,7 +14,7 @@
 import Nimble
 import XCTest
 
-@testable import RevenueCat
+@_spi(Internal) @testable import RevenueCat
```

Comment:
I don't think you need `@_spi(Internal)` here. Only in the places where you're actually going to use a `spi`-protected API